doc/reference/views.rst

```diff
@@ -1069,11 +1073,9 @@ take the following attributes:
 ``drag_resize``
   resizing of the tasks, default is ``true``
 
-.. ``progress``
-    name of a field providing the completion percentage for the record's event,
-    between 0 and 100
-.. consolidation_exclude
-.. consolidation_color
```

Comment:
And the consolidation_color? 🤔

Reply:
This attribute apparently existed at some point in odoo, but cannot be find in the .rng file https://github.com/odoo/odoo/blob/11.0/odoo/addons/base/rng/gantt_view.rng). 
And by looking in the js code the only place I can find it is here (https://github.com/odoo/enterprise/blob/11.0/web_gantt/static/src/js/gantt_renderer.js#L701) and here (https://github.com/odoo/enterprise/blob/11.0/web_gantt/static/src/js/gantt_renderer.js#L385) but it seems to refer to the task color; anyway it seems it is not a global gantt attribute anymore.